gamemodes/terrortown/gamemode/server/sv_corpse.lua

```diff
@@ -72,7 +68,13 @@ function CORPSE.SetCredits(rag, credits)
 	rag:SetDTInt(dti.INT_CREDITS, credits)
 end
 
-local function IdentifyBody(ply, rag)
+---
+-- Identifies the corpse, registers it and announces it to the server, if possible.
+-- @param Player ply The player that tries to identify the body
+-- @param Entity rag The ragdoll entity that is searched
+-- @param[default=0] number searchUID The unique search ID that is used to keep track of the search for the UI
```

Comment:
but how do I handle it behind the scenes? I want to pass it to the function and a global variable wont work because the UID is transmitted via netnessages. It's basically the same as it was in the old TTT, I only made the functions public and added documentation. And I refactored them of course